indico_toolkit/indico_wrapper/modelop.py

```diff
@@ -0,0 +1,146 @@
+from indico import IndicoClient
+from indico.queries import GraphQLRequest
+import json
+import string
+
+
+class ModelOp:
+
+    """
+    Class to assit in Model Group Operation calls.
+    """
+
+    def __init__(self, client: IndicoClient):
+        self.client = client
+
+    def get_model_options(
+        self, model_group_id: int, model_id: int | None = None
+    ) -> dict[str, object]:
+        """
+        Return model options for `model_id` of `model_group_id`
+        or the most recent model if `model_id` is not specified.
+        Args:
+            model_group_id (int): id of model group
+            model_id (int, optional): argument to return a specific model within a model group
+        """
+        all_model_options = self.get_all_model_options(model_group_id)
+
+        if model_id is None:
+            return next(all_model_options)
+
+    
+        for model_options in all_model_options:
+            if model_options["id"] == model_id:
+                return model_options
+        else:
+            raise RuntimeError(
+                f"Model group {model_group_id} does not have a model with ID {model_id}"
+            )
+
+    def get_all_model_options(self, model_group_id: int) -> Iterator[dict[str, object]]:
```

Comment:
We'll need to import `Iterator`. Otherwise the module has a `NameError` at runtime.